src/Tools/dotnet-counters/CounterMonitor.cs

```diff
@@ -475,7 +476,8 @@ private void StopMonitor()
                 using (DiagnosticsClientHolder holder = await builder.Build(ct, _processId, diagnosticPort, showChildIO: false, printLaunchCommand: false))
                 using (VirtualTerminalMode vTerm = VirtualTerminalMode.TryEnable())
                 {
-                    bool useAnsi = vTerm.IsEnabled;
+                    bool useWindows = !RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
+                    bool useAnsi = vTerm.IsEnabled && useWindows;
```

Comment:
```suggestion
                    bool useAnsi = vTerm.IsEnabled && !RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
```

or rename the variable to `isNonWindows` or similar.

Reply:
Done